docs/organizational/contribute/contribute_to_translation.txt

```diff
@@ -89,11 +89,41 @@ Only for transifex maintainers
 
 .. note:: This section is only for the maintainers of a transifex group!
 
+For main Geonode localization files
+----------------------------------
+
+These files are generated using standard django-admin tools : makemessages and compilemessages command line tools.
+
+At first make sure to install transifex client and configure your .transifexrc with our credentials as::
+
+   [https://www.transifex.com]
+   username = YOURUSERNAME
+   token =
+   password = PUTYOURPW
+   hostname = https://www.transifex.com
+
+Then to prepare the PR in our branch, execute following steps on your system::
+
+   $ # go to the [yourpath]/geonode/geonode/ subdirectory
```

Comment:
You should remove the comments here and maybe using them updating the explanation of what is going on